index.d.ts

```diff
@@ -111,7 +111,7 @@ export interface UseTableInstanceProps<D extends object> {
   rows: Row<D>[]
   getTableProps: (props?: object) => object
   getTableBodyProps: (props?: object) => object
-  prepareRow: (row: Row<D>) => void
+  prepareRow: (row: Row<D>) => unknown
```

Comment:
Maybe this could be narrowed?
The docs on what can be return leave this pretty open.
https://github.com/tannerlinsley/react-table/blob/master/docs/api.md#instance-properties

Reply:
This was discussed in my PR. Instead of changing this, the example should change its code.

```ts
(row, i) => {
  prepareRow(row)
  return (
    <tr {...row.getRowProps()}>
      {row.cells.map(cell => (
        <td {...cell.getCellProps()}>{cell.render('Cell')}</td>
      ))}
    </tr>
  )
}
```

Reply:
for reference, discussion was at https://github.com/tannerlinsley/react-table/pull/1535#discussion_r330128358

> the example should change its code

I'll revert 94bd828 and apply an update to the example code.

> Or modify the example's config to not throw errors on that

When it comes to a testing with strict vs loose/flexible config, I'd prefer to keep it strict.
It makes it easier for downstream adopters to use the test code as a reference for their own code.